test/manual/definitions/apm/node/node-linux2.json

```diff
@@ -17,7 +17,7 @@
   "services": [
     {
       "id": "nodetron",
-      "source_repository": "https://github.com/newrelic/demo-nodetron.git",
+      "source_repository": "-b replace_supervisord_with_pm2 https://github.com/newrelic/demo-nodetron.git",
```

Comment:
The branch 'replace_supervisord_with_pm2' doesn't exist anymore.  We should pull from head, even though head start node under Supervisord.

Reply:
👍